index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse</h1>
```

Comment:
- [ ] Kindly I recommend you to follow the best practices of HTML5 Syntax, For More information you can refer to this [document](https://www.w3schools.in/html5/syntax) Just Like this : 
```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <link rel="stylesheet" type="text/css" href="Style.css">
        <title></title>
        </head>
        <body>
            <h1 class=""></h1>
        </body>
        </html>
```

- [ ]  As stated the project requirement Kindly make sure that A .css file that includes `one class` with styles for your header (make it in your favorite color).
- ![Capture](https://user-images.githubusercontent.com/93598189/211501647-d22fcab1-eabd-46f6-84a6-f12672abe6f9.PNG)

Reply:
Fixed.